maintainers/maintainer-list.nix

```diff
@@ -10233,6 +10233,12 @@
     githubId = 347983;
     name = "Udo Spallek";
   };
+  ulrikstrid = {
```

Comment:
Creating a maintainer for someone else is questionable at best.

Reply:
Are you suggesting to leave the commit in #120532?

Reply:
Not suggesting anything, I just think this is another #120284.

Reply:
I remove the commit.